site/content/en/blog/releases/1.22.0-rc

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.21.0 - Lots of Bug Fixes!"
+linkTitle: "1.21.0"
+date: "2022-02-15"
+---
+This is the 1.22.0-rc release of Agones.
+
+This is the 1.22.0-rc release of Agones.
+
+This Release  fixes the issue with  agones-images on ARM systems and other bug fixes. 
```

Comment:
There are a couple of other changes worth mentioning here:

1. For multi-cluster allocation we have the GCP example (checked in) and also a link to a 3rd party EKS example
2. Cert manager integration